tools/perf/Makefile

```diff
@@ -0,0 +1,131 @@
+TOP := $(shell pwd)
+TOOLS_DIR = $(HOME)/bin
+CURL_PATH ?= curl
+CURL_DOWNLOAD := $(CURL_PATH) --location --fail --progress-bar
+
+KIND_CLUSTER_NAME ?= kuma
+KIND_KUBECONFIG_DIR := $(HOME)/.kube
+KIND_KUBECONFIG := $(KIND_KUBECONFIG_DIR)/kind-$(KIND_CLUSTER_NAME)-config
+HELM_PATH := $(TOOLS_DIR)/helm
```

Comment:
Shouldn't this be `?=` ? That way someone can have helm installed somewhere else. This applies to other tools as well (kind, kubectl).